news/_posts/2021-07-15-responsive-back-office.md

```diff
@@ -0,0 +1,147 @@
+--- 
+layout: post
+title: "Towards a more mobile-friendly back office"
+subtitle: "Quick wins for a responsive back office"
+date: 2021-07-28 18:00:00
+authors: [ScottCosio]
+icon: icon-mobile
+tags:
+ - 1.7.8
+ - version
+ - minor
+ - product
+ - ux
+---
+
+Until now, PrestaShop always relied on a desktop-first responsive design strategy. Indeed, there are few use cases on mobile. But you can't prevent users from checking their store's back office on their mobile!
+
+## Why work on responsive design?
+
+They were some issues that affected the user experience on mobile:
+
+- Difficulties in accessing important store information
+- Hidden content because of overlapping components
+- Text readability issues
+- Bugs and lack of consistency on display (e.g. not the same padding)
+
+All of these issues make it difficult for users to use PrestaShop on mobile devices. Also, they cause a bad reputation for the PrestaShop open source project.
+
+## First steps: quick wins 💪🏻
+
+[Valentin](https://www.linkedin.com/in/%F0%9F%9A%80-valentin-szczupak-0280aba1/) (front-end developer), and I, [Scott](https://www.linkedin.com/in/scottcosio/) (product designer), had a limited bandwidth to work on this project due to others features to work on. In order to avoid breaking changes, we only scoped small fixes on the mobile version of the backoffice a.k.a. quick wins.
```

Comment:
```suggestion
[Valentin](https://github.com/neomaking) (front-end developer), and I, [Scott](https://github.com/prestascott) (product designer), had a limited bandwidth to work on this project due to others features to work on. In order to avoid breaking changes, we only scoped small fixes on the mobile version of the backoffice a.k.a. quick wins.
```
I suggest linking to GitHub profiles as this allows people to follow your work on GitHub.